doctr/models/builder.py

```diff
@@ -51,6 +51,12 @@ def _sort_boxes(boxes: np.ndarray) -> np.ndarray:
             indices of ordered boxes of shape (N,)
         """
         if boxes.shape[1] == 5:
+            boxes = rotate_boxes(
+                loc_preds=np.concatenate((boxes, np.zeros((boxes.shape[0], 1))), -1),
+                angle=-np.median(boxes[:, 4]),
+                orig_shape=(1024, 1024),
+                min_angle=5.,
+            )
```

Comment:
That's good for line aggregation, but don't we need to rotate the line back afterwards then?

Reply:
I don't think so, because we only rotate the boxes to get the sorted idxs of "straight" boxes, but then we use those idxs to reconstruct the document with the original boxes (not rotated, the rotated ones are just used locally in resolve_lines)